torch_frame/data/multi_tensor.py

```diff
@@ -325,6 +324,49 @@ def select(
     def _single_index_select(self, index: int, dim: int) -> _MultiTensor:
         raise NotImplementedError
 
+    def fillna_(self, fill_value: Union[int, float, Tensor]):
+        """Fill the :obj:`MultiTensor` with fill_value.
+
+        Args:
+            index (int): A column index of the tensor to select.
+            fill_value (Union[int, float, Tensor]): Scalars or vector
+                tensor of size (num_cols,) or (1, num_cols) to replace
+                NaNs.
+        """
+        if isinstance(fill_value, Tensor):
+            if fill_value.dim(
+            ) != 0 and not fill_value.numel() == self.num_cols:
+                raise ValueError(
+                    "fillna_ with Tensor of shape "
+                    f"{fill_value.shape} is not supported. "
+                    f"Please use a Tensor of shape ({self.num_cols},)"
+                    f" or (1, {self.num_cols}).")
+            if fill_value.dim() != 0:
+                if fill_value.dim() == 1:
+                    fill_value = fill_value.unsqueeze(0)
```

Comment:
Not sure I get all this `dim` logic here. Why not just do `fill_value.view(-1)` at the start?